magclassic/static/theme/prereg.css

```diff
@@ -1,8 +1,15 @@
 .masthead {
-    height:325px;
     background-image:url("../theme/banner.png");
```

Comment:
There are some comments about the implementation of this in the [main uber PR](https://github.com/magfest/ubersystem/pull/2409)